docs/source/restore/restore-tables.rst

```diff
@@ -22,8 +22,34 @@ Prerequisites
    Note that an empty table is not necessarily truncated!
 
 * It is important to remember current mode of the `tombstone_gc <https://www.scylladb.com/2022/06/30/preventing-data-resurrection-with-repair-based-tombstone-garbage-collection/>`_ option.
-    In order to ensure data consistency, Scylla Manager alters restored tables options with ``tombstone_gc = {'mode': 'disabled'}``.
-    Right now, it's user responsibility to reset the ``tombstone_gc`` option as a part of the restore tables *follow-up action*.
+   In order to ensure data consistency, Scylla Manager alters restored tables options with ``tombstone_gc = {'mode': 'disabled'}``.
+   Right now, it's user responsibility to reset the ``tombstone_gc`` option as a part of the restore tables *follow-up action*.
+
+* It is strongly advised to drop all materialized views and secondary indexes that are built on top of the table that is about to be restored. They must be recreated after the restore.
+   We do not guarantee that materialized views and secondary indexes will be restored correctly.
+   Materialized views and secondary indexes must be recreated after the table is successfully restored.
+   Please find some useful CQL commands below:
```

Comment:
I sugget to reword lines 28-31 to:
".. note::
     Materialized views and secondary indexes must be recreated after the table is successfully restored."

No need to advice or to (not) guarantee. Just list the steps

Reply:
OK, 28-31 is replaced with just "Materialized views and secondary indexes must be recreated after the table is successfully restored." as suggested.